phsa-username-updater/src/main/java/KeycloakUsernameUpdater.java

```diff
@@ -11,15 +11,20 @@
 public class KeycloakUsernameUpdater {
 
     // Keycloak configuration
+    private static final String KEYCLOAK_URL = "https://common-logon.hlth.gov.bc.ca/auth";
+    private static final String REALM = "moh_applications";
+    private static final String CLIENT_ID = "DAVID_SHARPE_DELETE_PROD_20250210.1650";
+    private static final String CLIENT_SECRET = System.getenv("DAVID_SHARPE_DELETE_PROD_20250210.1650");
```

Comment:
These are unrelated updates, but please approve them too. They are local changes for a now completed RFC.